activation/activation.go

```diff
@@ -405,7 +406,11 @@ func (b *Builder) BuildNIPostChallenge(ctx context.Context, nodeID types.NodeID)
 	case err != nil:
 		return nil, fmt.Errorf("get nipost challenge: %w", err)
 	case challenge.PublishEpoch < current:
-		// challenge is stale
+		logger.Info(
+			"existing NiPOST challenge is stale, resetting state",
+			zap.Stringer("current_epoch", current),
```

Comment:
i think it is better to use zap.Uint32 . it is cheaper to encode and you can also do filtering (>, <) in loki